lib/beaker/host/unix.rb

```diff
@@ -42,9 +44,11 @@ def self.foss_defaults
         'puppetserver-confdir' => '/etc/puppetserver/conf.d',
         'puppetservice'     => 'puppetmaster',
         'puppetpath'        => '/etc/puppet',
+        'puppetconfdir'     => '/etc/puppet',
         'puppetvardir'      => '/var/lib/puppet',
         'puppetbin'         => '/usr/bin/puppet',
         'puppetbindir'      => '/usr/bin',
+        'systembindir'      => '/usr/bin',
```

Comment:
The `aio_defaults` are an override of `foss_defaults`, so it gets that value for free